sentry-rails/lib/sentry/rails/capture_exceptions.rb

```diff
@@ -32,7 +32,9 @@ def capture_exception(exception, env)
           current_scope.set_transaction_name(original_transaction)
         end
 
-        Sentry::Rails.capture_exception(exception)
+        Sentry::Rails.capture_exception(exception).tap do |event|
+          env[ERROR_EVENT_ID_KEY] = event.event_id if event
```

Comment:
this should also go into the rack middleware [here](https://github.com/getsentry/sentry-ruby/blob/master/sentry-ruby/lib/sentry/rack/capture_exceptions.rb#L57) ?